public/src/components/game.js

```diff
@@ -69,7 +69,12 @@ export default React.createClass({
     let startControls = d.div({},
       d.div({}, `Format: ${App.state.format}`),
       LBox('bots', 'bots'),
-      LBox('timer', 'timer'),
+      d.div({}, d.input({
```

Comment:
There is a `num-lands` CSS class that shortens the textbox width. It might be a good idea to refactor that into a general-purpose number text box class and use it here, since the textbox is rather wide.